src/components/Dog/CardDogTable.jsx

```diff
@@ -35,7 +35,16 @@ function DogCard({ className, dog, onClick }) {
       onClick={onClick || (() => {})}
     >
       <div className="flex w-[240px] h-[240px] items-center justify-center rounded-lg bg-primary-gray overflow-hidden">
-        {dog.image ? <Image alt={dog.name} width={350} height={350} src={dog.image} />
+        {dog.image ? 
```

Comment:
is there a reason you are using regular styles instead of tailwind? I would prefer you to use tailwind to keep consistency throughout the codebase